org.eclipse.xtext.xbase.ui/xtend-gen/org/eclipse/xtext/xbase/ui/editor/XbaseEditorInputRedirector.java

```diff
@@ -47,7 +47,7 @@
   public ITypeRoot getTypeRoot(final IEditorInput it) {
     ITypeRoot _xblockexpression = null;
     {
-      final Object adapter = it.getAdapter(IJavaElement.class);
```

Comment:
are you sure that works with luna tp?

Reply:
hmm, will double check

Reply:
I thought I had a Luna TP enabled, but now set up a new devenv with Luna TP. Actually the generics are not present on Luna, makes this one invalid. I'll close it.